### PR TITLE
 feat(st-form)[UX-58, UX-51]: Adapt it to display checkboxes, integer and number inputs and add tooltips to checkboxes

### PR DESCRIPTION
**Pull request by emgaitan-stratio**
Closes #UX-58 and #UX-51
### Documentation
- [x] Add the issue in PR description
- [x] Have changelog updated
- [x] You fill the milestone value
- [x] You add some label
- [x] Have example running in demo app
- [x] Review id on demo is the same of egeo folder

### Code
- [x] Pass Sass lint task
- [ ] Have qaTags

### Tests
- [ ] Have at least 70% tests coverage

### CODE REVIEW

File: src/egeo-demo/st-form-demo/st-form-demo.ts [hunk 1]
```diff
@@ -52,4 +54,15 @@ export class StFormDemoComponent {
    onChange(model: any): void {
       this.model = model;
    }
+
+   disableForm(): void {
+      if (this.formModel.control.enabled) {
+         this.formModel.control.disable();
+      } else {
+         console.log('eee')
```
Comment (fjsc) [inline src/egeo-demo/st-form-demo/st-form-demo.ts]: Remove this console.log

File: src/egeo-demo/st-form-demo/st-form-demo.ts [hunk 2]
```diff
@@ -52,4 +54,14 @@ export class StFormDemoComponent {
    onChange(model: any): void {
       this.model = model;
    }
+
```
Comment (acuevas-stratio) [inline src/egeo-demo/st-form-demo/st-form-demo.ts]: The name of this method should be something like "toggleForm". 
On the other hand, markForCheck() is not needed when disabled?

File: src/lib/st-form/st-form.component.ts [hunk 3]
```diff
@@ -102,8 +97,13 @@ export class StFormComponent implements ControlValueAccessor, OnInit {
       this.onTouched = fn;
    }
 
-   // Allows Angular to disable the list.
-   setDisabledState(isDisabled: boolean): void {
+   // Allows Angular to disable the form.
```
Comment (acuevas-stratio) [inline src/lib/st-form/st-form.component.ts]: Again, this method not only disables, but also toggle the state

Comment (emgaitan-stratio) [reply src/lib/st-form/st-form.component.ts]: This is a needed method for ControlAttributeAccesor interface

**Pull request merged**